server/app/assets/stylesheets/uswds/_uswds-theme.scss

```diff
@@ -0,0 +1,20 @@
+/*
+----------------------------------------
+USWDS with settings overrides
+----------------------------------------
+Customize USWDS styles here or prevent USWDS utilities (custom HTML classes
+scoped to a single CSS property) from being applied by setting 'output' to 'false'.
+See https://designsystem.digital.gov/documentation/settings/ and
+https://designsystem.digital.gov/utilities/.
+*/
+
+@use 'uswds-core' with (
+  $theme-font-path: '../../../../node_modules/@uswds/uswds/dist/fonts',
```

Comment:
When the Webpack `sass-loader` compiles Sass into CSS, it doesn't update the font and image urls to be relative to the Sass entry point.  For us, that's `app/assets/stylesheets/uswds/styles.scss` [sass-loader documentation](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/#loading-uswds-in-your-sass-entry-point-2).  The fonts and images are still in the uswds directory under node_modules.  So, I specify the correct paths here in the uswds settings.

Reply:
Please move this github comment into a code comment here.